geekdanitek/questioner-react
# #166402479 User can see question and comments

#### Description
This PR enables the user to see a question and comments

#### Type of change
- Non-breaking change

#### How Has This Been Tested?
- clone this branch `story-166402479-see-question`
- cd in the application folder
- run `npm start:dev`
- go to `/meetups/:meetupdId/question/:questionId`

#### Checklist:
- [x] Users can see a question and comments page

#### Story (PT)
[#166402479](https://www.pivotaltracker.com/n/projects/2333428/stories/166402479)
